apps/api-documenter/src/documenters/DocumenterConfig.ts

```diff
@@ -23,9 +24,7 @@ export class DocumenterConfig {
   /**
    * The JSON Schema for API Documenter config file (api-documenter.schema.json).
    */
-  public static readonly jsonSchema: JsonSchema = JsonSchema.fromFile(
-    path.join(__dirname, '..', 'schemas', 'api-documenter.schema.json')
-  );
+  public static readonly jsonSchema: JsonSchema = JsonSchema.fromLoadedObject(apiDocumenterSchema);
```

Comment:
@iclanton  To optimize performance, `JsonSchema.fromFile()` does not actually load or parse the schema until it is used to validate something.

Does your change make the performance worse?

Reply:
We probably shouldn't be initializing instances of `JsonSchema` if we aren't going to use them.

Perf may be slightly worse if the tool isn't bundled and the schema is never used, but this also validates at compile-time that the path to the schema is correct.

Reply:
From some further investigation, this shouldn't regress performance.

Reply:
Loading and parsing JSON is very, very cheap.